### PR TITLE
Add api method 'getpaymentmethods'

### PR DESCRIPTION
**Pull request by ghubstan**
Returns a list of supported payment method IDs.  This gives CLI users the correct payment method id for creating a new payment account.

- Added gRPC service `GetPaymentMethods`.

- Added gRPC boilerplate method to `GrpcPaymentAccountsService`.

- Added implimentation to `CoreApi` -> `CorePaymentAccountsService`.

- Refactored PaymentAccount test hierarchy.

- Add api method to CLI.

This is the 13th in a chain of PRs beginning with https://github.com/bisq-network/bisq/pull/4793.  

PR https://github.com/bisq-network/bisq/pull/4808 should be reviewed before this one.

### CODE REVIEW

**Comment by ghubstan** [comment]
Backing out the changes made to `BsqSendView` in commit 446bd32 does not affect the `sendbsq` functionality in the api or desktop, but the small bit of duplication resulting from this single file revert should be removed in a later PR.

File: apitest/src/test/java/bisq/apitest/method/payment/GetPaymentMethodsTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test
+    @Order(1)
+    public void testGetPaymentMethods() {
+        List<String> paymentMethodIds = getPaymentMethods(alicedaemon)
+                .stream()
+                .map(p -> p.getId())
+                .collect(Collectors.toList());
+        assertTrue(paymentMethodIds.size() > 25);
```
Comment (sqrrm) [inline apitest/src/test/java/bisq/apitest/method/payment/GetPaymentMethodsTest.java:46]: This seems like a fragile test as we continuously change the paymentMethods

**Review by sqrrm — COMMENTED** [review]
utACK

File: apitest/src/test/java/bisq/apitest/method/payment/GetPaymentMethodsTest.java (hunk 1 above)
Comment (ghubstan) [reply apitest/src/test/java/bisq/apitest/method/payment/GetPaymentMethodsTest.java:46]: Assert the payment method list size is >= 20?

Comment (ghubstan) [reply apitest/src/test/java/bisq/apitest/method/payment/GetPaymentMethodsTest.java:46]: Fixed in commit 2989f1be3f6dd9990d48642ac252bd350d906dc7.

**Comment by ghubstan** [comment]
This PR is being closed after requested changes were committed in a the final, downstream PR (https://github.com/bisq-network/bisq/pull/4858) in this chain.

Merging of the final PR in the chain implies approval and merging of this PR.

**Pull request closed without merging**